src/main/scala/com/twitter/finagle/postgres/PostgresClient.scala

```diff
@@ -69,7 +69,7 @@ object PostgresClient {
 
   case class TypeSpecifier(receiveFunction: String, typeName: String, elemOid: Long = 0)
 
-  private[finagle] val defaultTypes = Map(
+  val defaultTypes = Map(
```

Comment:
Reading these comments again, I think my tone may be slightly abrasive. Sorry about that!